### PR TITLE
chore: update build and release processes

### PR DESCRIPTION
**Pull request by eliseeborn**
mostly just checked what @haxxmaxx had done in action button

### CODE REVIEW

**Comment by haxxmaxx** [comment]
You need to add the missing environment variables to circle. You can import them from another project if you have access to that (I think you need to be admin). Have a look otherwise I can help you

**Comment by eliseeborn** [comment]
> You need to add the missing environment variables to circle. You can import them from another project if you have access to that (I think you need to be admin). Have a look otherwise I can help you

I don't seem to have the right access, so some help would be nice :)

**Comment by haxxmaxx** [comment]
The only one missing was the NPM_TOKEN so I added that. I thought that was why it was failing, but it is something else

File: package.json [hunk 1]
```diff
@@ -71,13 +77,15 @@
     "eslint-config-airbnb-base": "14.2.1",
     "eslint-plugin-import": "2.25.4",
     "eslint-plugin-mocha": "9.0.0",
+    "fs-extra": "^10.0.0",
```
Comment (cbt1) [inline package.json:80]: Should be looked to version `10.0.0`

File: package.json [hunk 2]
```diff
@@ -71,13 +77,15 @@
...
     "husky": "4.3.8",
     "jest": "^27.4.7",
     "jest-junit": "^13.0.0",
     "less": "^4.1.2",
     "lint-staged": "11.2.6",
     "mocha-junit-reporter": "2.0.2",
     "shx": "0.3.4",
+    "yargs": "^17.2.1",
```
Comment (cbt1) [inline package.json:88]: Should be looked to version `17.1.1`. Don't forget to run `yarn` after so that the `yarn.lock` file gets updated.

**Comment by haxxmaxx** [comment]
The files in the `tools` dir are copy pasted right?

**Comment by eliseeborn** [comment]
> The files in the `tools` dir are copy pasted right?

yes

**Review by haxxmaxx — APPROVED** [review]
LGTM

**Review by cbt1 — APPROVED** [review]

**Pull request merged**